.ci/jenkins_job_build.sh

```diff
@@ -185,7 +185,9 @@ esac
 # run the tests.
 if command -v kata-runtime; then
 	echo "Logging kata-env information:"
-	kata-runtime kata-env
+	#kata-runtime kata-env
+	echo "CI is broken, not running kata-env"
```

Comment:
mmmm  CI is not broken ...  `kata-runtime kata-env` is not working ?

Reply:
it is broken because kata-env is not working :P

Reply:
This help me to find the bug :)